cli/src/main/java/org/jboss/as/cli/impl/aesh/SynopsisGenerator.java

```diff
@@ -180,8 +181,8 @@ private String addSynopsisOption(SynopsisOption currentOption) {
             }
             synopsisBuilder.append(" |");
             // Keep the set of all conflicts, they will be removed from all conflicts, being handled at this level.
-            Set<SynopsisOption> conflicts = new HashSet<>(currentOption.conflictWith);
-            Set<SynopsisOption> conflicts2 = new HashSet<>(currentOption.conflictWith);
+            Set<SynopsisOption> conflicts = new LinkedHashSet<>(currentOption.conflictWith);
+            Set<SynopsisOption> conflicts2 = new LinkedHashSet<>(currentOption.conflictWith);
```

Comment:
I think that the correct fix would be to use 
```
Set<SynopsisOption> conflicts2 = new TreeSet<>(Comparator.comparing(SynopsisOption::getName));
conflicts2.addAll(currentOption.conflictWith);
```
In order to iterate the conflicts using the natural ordering of Options (name).